app/controllers/dresses_controller.rb

```diff
@@ -6,4 +6,33 @@ def index
   def show
     @dress = Dress.find(params[:id])
   end
+
+  def new
+    @dress = Dress.new
+  end
+
+  def create
+    @dress = Dress.new(dress_params)
+    @dress.user = current_user
+    if @dress.save
+      redirect_to dress_path(@dress)
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @dress = Dress.find(params[:id])
+    flash[:remove] = "\"#{@dress.brand} dress\" deleted"
```

Comment:
should this go after the dress is actually deleted?

Reply:
hmm good question. if we delete the dress beforehand, can we still get it's dress.brand info?